Assets/Scripts/Environment/Hangar/Battery/BatterySlot.cs

```diff
@@ -1,79 +1,79 @@
-using System.Text;
-using UnityEngine;
-
-public class BatterySlot : InteractableObject
-{
-    [SerializeField] private HangarSlot hangarSlot;
-
-    public void TransferEnergyToShip(Battery battery)
-    {
-        bool canTransferEnergy = CanTransferEnergy(battery);
-
-        if (canTransferEnergy)
-        {
-            ShipsManager.EnableWarp(hangarSlot.Ship);
-            battery.Discharge();
-        }
-        
-        LogInformation(battery, canTransferEnergy);
-    }
-
-    private bool CanTransferEnergy(Battery battery)
-    {
-        return hangarSlot != null
-            && hangarSlot.Ship != null
-            && battery.IsCharged
-            && !hangarSlot.Ship.CanWarp;
-    }
-
-    private void OnTriggerStay(Collider other)
-    {
-        if (IsPlayerColliding
-            && Input.GetKey(PlayerConstants.ActionKey))
-        {
-            Battery battery = other.GetComponentInChildren<Battery>();
-            if (battery != null)
-            {
-                TransferEnergyToShip(battery);
-            }
-        }
-    }
-

```

Comment:
I think your line endings are causing this. It's not a big deal, but this command should stop the line endings changing:
 
``git config --global core.autocrlf false``

Makes it a bit easier to review if line endings are the same.